api/tests/integration/infrastructure/repositories/campaign-repository_test.js

```diff
@@ -203,58 +202,55 @@ describe('Integration | Repository | Campaign', function () {
     let campaign;
 
     beforeEach(function () {
-      const bookshelfCampaign = databaseBuilder.factory.buildCampaign({
-        id: 1,
-        title: 'Title',
-        customLandingPageText: 'Text',
-        archivedAt: new Date('2019-03-01T23:04:05Z'),
-        ownerId: databaseBuilder.factory.buildUser({ id: 1 }).id,
-      });
-      campaign = domainBuilder.buildCampaign(bookshelfCampaign);
+      campaign = databaseBuilder.factory.buildCampaign();
+
       return databaseBuilder.commit();
     });
 
     it('should return a Campaign domain object', async function () {
       // when
-      const campaignSaved = await campaignRepository.update(campaign);
+      const campaignSaved = await campaignRepository.update({ id: campaign.id, title: 'Title' });
 
       // then
       expect(campaignSaved).to.be.an.instanceof(Campaign);
     });
 
     it('should not add row in table "campaigns"', async function () {
       // given
-      const rowCount = await BookshelfCampaign.count();
-
+      const rowsCountBeforeUpdate = await knex.from('campaigns');
       // when
-      await campaignRepository.update(campaign);
+      await campaignRepository.update({ id: campaign.id, title: 'Title' });
 
       // then
-      const rowCountAfterUpdate = await BookshelfCampaign.count();
-      expect(rowCountAfterUpdate).to.equal(rowCount);
+      const rowCountAfterUpdate = await knex.from('campaigns');
```

Comment:
```suggestion
      const rowCountAfterUpdate = await knex.from('campaigns').count();
```
c'est pas moins lourd ?

Reply:
Pas forcement parce que  quand tu fais 
`const rowCountAfterUpdate = await knex.from('campaigns').count();` 
le résultat c'est `rowCountAfterUpdate = [ { count : 1 }]`
donc tu es obligé de faire 
>  const rowCountAfterUpdate = await knex.from('campaigns').count().first();
      expect(rowCountAfterUpdate.count).to.equal(rowsCountBeforeUpdate.count);
      
ou tu destructures
>  const { count: rowCountAfterUpdate } = await knex.from('campaigns').count().first();
      expect(rowCountAfterUpdate).to.equal(rowsCountBeforeUpdate);